db/migrate/20230414091257_rename_append_domain_setting.rb

```diff
@@ -0,0 +1,7 @@
+class RenameAppendDomainSetting < ActiveRecord::Migration[6.1]
+  def change
+    original_value = Setting.find_by(name: 'append_domain_name_for_hosts').value
+    Setting.find_by(name: 'display_fqdn_for_hosts').update(value: original_value)
+    Setting.find_by(name: 'append_domain_name_for_hosts').delete
```

Comment:
`Setting.find_by(name: 'append_domain_name_for_hosts').update(name: 'display_fqdn_for_hosts')`

Reply:
This is pretty weird, update method failed when updating name but not value. In the end `update_attribute` worked.